bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v2/ImageImplTest.java

```diff
@@ -56,6 +56,21 @@ public void testExportedType() {
         assertEquals(ImageImpl.RESOURCE_TYPE, image.getExportedType());
     }
 
+    /*
+    @Test
```

Comment:
@Samyak1080 - do you plan to reenable this test case?

Reply:
What code do we need in wcm-io-testing?

Reply:
https://github.com/wcm-io/wcm-io-testing/blob/develop/aem-mock/src/main/java/io/wcm/testing/mock/aem/MockAsset.java#L215
As explained in PR description, this implementation of getID() method did not exist before. Now that it has been implemented, it will be included in the upcoming released version 2.2.16 of AEM Mocks. Once this version is released, we can update the dependency in our POM and re-enable this test case.

Reply:
Thanks for clarifying @Samyak1080!

Reply:
Unfortunately the update version of AEM Mocks (2.2.16) doesn't return a proper `jcr:uuid`. I created https://wcm-io.atlassian.net/browse/WTES-33 for this issue